app/assets/javascripts/selectize_config.js

```diff
@@ -1,6 +1,15 @@
 $(function(){
-  $('.selectize-container').selectize({
+  $select = $('.selectize-container').selectize({
     plugins: ['remove_button'],
     closeAfterSelect: true
   });
+
+ $('.page-filter__reset').click( function() {
```

Comment:
It would be nice to move this to its own file and use the pubsub system to only activate it on the relevant page, ie
`$.publish('search:load')` on the slim template and then make `javascripts/search.js` with this code and the datatable init and make them get called when the event gets triggered.